app/templates/components/feed-item.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="article-preview">
+    <div class="article-meta">
+        <a href="profile.html"><img src= {{imageSrc}} /></a>
+        <div class="info">
+            <a href="" class="author">{{author}}</a>
+            <span class="date">{{date}}</span>
+        </div>
+        <button class="btn btn-outline-primary btn-sm pull-xs-right">
+            <i class="ion-heart"></i> {{hearts}}
+        </button>
+    </div>
+    <a href="" class="preview-link">
+        {{content}}
+        {{yield}}
```

Comment:
I added this here incase we would want something after the content. It can be ignored and kept here or we can remove it.